sae_lens/training/config.py

```diff
@@ -106,6 +106,10 @@ class LanguageModelSAERunnerConfig:
 
     dead_feature_threshold: float = 1e-8
 
+    # Evals
+    n_eval_batches: int = 10
+    n_eval_seqs: int | None = None  # useful if evals cause OOM
```

Comment:
nit: is this the batch size used for eval? If so, this would be clearer named as `eval_batch_size` to match the naming of `store_batch_size` and `train_batch_size` params

Reply:
ah looks like I must have commented as this was being merged 😅. Not a big issue regardless

Reply:
True. Feel free to make a PR for this.